superset-frontend/spec/helpers/testing-library.tsx

```diff
@@ -81,8 +95,8 @@ export function createWrapper(options?: Options) {
       result = <BrowserRouter>{result}</BrowserRouter>;
     }
 
-    if (useQuery) {
-      result = <QueryProvider>{result}</QueryProvider>;
+    if (useApi && !useRedux) {
```

Comment:
What's the difference between `useRedux` and `useApi`? Looking at the code, the middleware and api reducer are also set when `useRedux` is true. Can we consolidate and use only `useRedux`?

Reply:
@michael-s-molina sounds good. I updated with `useRedux`
(yeah it looks like no longer needed since this option for react-query wrapper)